### DISCUSSION TITLE
Apache Impala Support

### DISCUSSION THREAD

**Discussion by Khalid-Nowaf** [beltran/gohive#234] [Q&A] [answered]
Can gohive connect to Impala demon since Impala utilized HiveServer2 over HTTP or the TCP binary?

**Comment by beltran** [comment]
Hello, I'm not aware of gohive ever having been used to connect to Impala.

**Comment by airosa** [comment, marked as answer]
Hello ! We are connecting to both Hive and Impala with gohive.

**Reply by Khalid-Nowaf** [reply]
This is an excellent news to me! Did you face any challenges?

**Comment by airosa** [comment]
In our setup (=cdp-pc) we are running multiple impala/hive instances behind a load balancing proxy and we are using HTTP. In this setup it is critical that the HTTP headers are processed correctly in gohive because the proxy uses the headers to pass the request to the correct impala/hive instance.
